### ISSUE TITLE
10250 port is not open in Windows RS3 firewall, in azure cni mode

### ISSUE THREAD

**Issue by yuedai** [Azure/acs-engine#2286]
**Is this a request for help?**:
Help
---

**Is this an ISSUE or FEATURE REQUEST?** (choose one):
Issue
---

**What version of acs-engine?**:
0.13.0
---

**Orchestrator and version (e.g. Kubernetes, DC/OS, Swarm)**
0.13.0

**What happened**:
Provisioned a kubernetes cluster with networkpolicy:azure. "Kubectl exec -it" does not work, reports could not connect to 10250 port of the agent node from private IP.

I login to the agent node VM and found the 10250 tcp port is blocked by firewall. After adding a rule on that machine, the issue is resolved on that node.

I don't know if the fix is the correct way, but please take a look to help fix the kubectl exec -it hang issue. 

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

**Comment by jackfrancis** [comment]
@saiyan86 Does this sound familiar in your testing w/ RS3?

**Comment by yuedai** [comment]
Besides... I am keeps on getting this warning in "Services" object:

"Error creating load balancer (will retry): error getting LB for service kube-system/kube-dns: Service(kube-system/kube-dns) - Loadbalancer not found"

Maybe I should open a new issue?
